README.md

```diff
@@ -58,6 +58,22 @@ The cosign verification of Rancher Prime images can be done with:
 slsactl verify <prime_image>:<tag>
 ```
 
+### Troubleshooting
+
+#### permission denied failures
+
+This application runs with strict landlock rules, which limits its
+permissions in the host system. Supporting Kernels will enforce such
+rules which may result in permission denied failures.
+
+When users that the specific access is valid (e.g. use of `docker-credential-helpers`)
```

Comment:
Just a small comment that I couldn't understand this phrase, seems like a word is missing perhaps?